soc/riscv/riscv-ite/common/encoding.h

```diff
@@ -1,8 +1,12 @@
 /*
- * Copyright (c) 2020 ITE Corporation. All Rights Reserved.
- *
- * SPDX-License-Identifier: Apache-2.0
- *
+ * Copyright and related rights are licensed under the Solderpad Hardware
```

Comment:
Is it clear this file needs its license changed?

Reply:
yep, AFAIU it is fine keep the license, the case is just add the original copyright

Reply:
The original is a modified Apache 2, then I prefer to just put Solderpad Hardware.